### DISCUSSION TITLE
Auth values not showing up in debugger, cookie helper can't find them either

### DISCUSSION THREAD

**Discussion by cheetowarrior** [UltimaHoarder/UltimaScraper#1243] [Q&A]
Hey, I got an older version of this to work a long while back, but now I'm trying to get it going again. When I log in and search the network debugger for '?app" though, nothing comes up. I tried installing the cookie helper extension, and clicking on it just says "Could not find valid cookie values, make sure you are logged into OnlyFans.". I am logged in, I tried logging in and out, I can access the profiles I'm trying to scrape, all that stuff. And if I try to use the browser login method, it just says "There's nothing to scrape" and starts over.

No idea why I can't locate the cookie values but I tried this in both Brave and Firefox... anyone have any idea what's going on?

**Comment by lEtranger1942** [comment]
Read documentation
